incident/tests/test_get_serialized_filters.py

```diff
@@ -41,32 +41,6 @@ def test_serialize_general__search_only(self):
             },
         ])
 
-    def test_serialized_filters_include_legal_order_type(self):
-        category = CategoryPageFactory(incident_filters=[
-            'legal_order_type',
-        ])
-        general, category = get_serialized_filters()
-        legal_order_type_filter = category['filters'][0]
-        self.assertEqual(legal_order_type_filter, {
-            'name': 'legal_order_type',
-            'title': 'Legal order type',
-            'type': 'choice',
-            'choices': LegalOrderType.choices
-        })
-
-    def test_serialized_filters_include_legal_order_status(self):
-        category = CategoryPageFactory(incident_filters=[
-            'legal_order_status',
-        ])
-        general, category = get_serialized_filters()
-        legal_order_type_filter = category['filters'][0]
-        self.assertEqual(legal_order_type_filter, {
-            'name': 'legal_order_status',
-            'title': 'Legal order status',
-            'type': 'choice',
-            'choices': LegalOrderStatus.choices
-        })
-
```

Comment:
I don't think we want to remove either of these tests, as both `legal_order_status` and `legal_order_type` are filters we want to preserve.